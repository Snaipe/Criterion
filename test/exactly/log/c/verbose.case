[act]

% log.c.bin --verbose

[assert]

stderr
  -transformed-by replace_version
  equals -contents-of expected/verbose.txt
