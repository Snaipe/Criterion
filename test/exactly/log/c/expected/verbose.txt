[----] Criterion <VERSION>
[====] Running 2 tests from logging:
[RUN ] logging::format
[----] Log messages are printf-formatted strings.
[PASS] logging::format
[RUN ] logging::simple
[----] This is an informational message. They are not displayed by default.
[WARN] This is a warning. They indicate some possible malfunction or misconfiguration in the test.
[ERR ] This is an error. They indicate serious problems and are usually shown before the test is aborted.
[PASS] logging::simple
[====] Synthesis: Tested: 2 | Passing: 2 | Failing: 0 | Crashing: 0 
